easybuild/easyconfigs/q/Qt/Qt-4.8.6.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Qt'
+version = '4.8.6'
+
+homepage = 'http://qt-project.org/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [ 'http://download.qt.io/official_releases/qt/4.8/%(version)s' ]
+sources = [ 'qt-everywhere-opensource-src-%(version)s.tar.gz' ]
+
+configopts = ' -opensource -nomake tests -nomake examples -confirm-license -release -shared -no-accessibility -no-sql-mysql -no-sql-sqlite'
```

Comment:
@jgphpc lines should not be longer than 120 characters, please break it up:

```python
configopts = "-opensource -nomake tests -nomake examples -confirm-license -release -shared "
configopts += " -no-accessibility -no-sql-mysql -no-sql-sqlite"
````